### DISCUSSION TITLE
Welcome to yahoofantasy discussions!

### DISCUSSION THREAD

**Discussion by mattdodge** [mattdodge/yahoofantasy#11] [Announcements]
## 👋 Hey fellow fantasy nerds! 🏆 
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

I was inspired to turn this on after a question raised by a user in #10. It's not an issue, but it's a totally valid question that others may have too, perfect for a discussion!

Use discussions to ask for general guidance, share interesting insights/scripts from your own fantasy leagues, or just chat about fantasy hacking. See ya there!

**Comment by Ace7866** [comment]
I cannot wait until I figure out my issues and I start pulling some raw data using your library! My league mates are gonna be scratching their heads for sure. But, as the league "historian", it seems my duty to up my analysis game. Plus it's tough to find historical data on Yahoo Fantasy's actual website.

**Reply by Ace7866** [reply]
Woohoo - I figured out the PATH issue! Now to see if I can actually wrap my head around the rest of the steps and library itself...
